pandas/tests/test_panel.py

```diff
@@ -1519,6 +1519,50 @@ def test_ffill_bfill(self):
         assert_panel_equal(self.panel.bfill(),
                            self.panel.fillna(method='bfill'))
 
+    def test_ffill_bfill_axis(self):
+        # GH 14499
+        a = Panel({
+            'a': np.arange(15, dtype=float).reshape((5, 3)),
+            'b': np.arange(15, 30, dtype=float).reshape((5, 3))
+        })
+        f0 = a.copy()
+        f1 = a.copy()
+        f2 = a.copy()
+        b0 = a.copy()
+        b1 = a.copy()
+        b2 = a.copy()
+        a['a'].loc[1, 1] = np.nan
```

Comment:
set using full ``.loc``, not chained indexing, e.g. ``f0.loc['a', 1, 1] = np.nan``